<?xml version="1.0" encoding="UTF-8"?>

<launch>
	
    <arg name="HEADWAY_SCALE" default="100.0"/>
    <arg name="SPEED_SCALE" default="40.0"/>
    <arg name="ego_veltopic" default="/vel"/>
    <arg name="relative_veltopic" default="/rel_vel"/>
    <arg name="headway_topic" default="/lead_dist"/>
    <param name="model" type="string" value="$(find onnx2ros)/../controllers/12Jul21_RL_vdes_output_cra/12Jul21_RL_vdes_output_cra.onnx"/>
    <param name="mode" type="string" value="prompt"/>
    <param name="v" type="double" value="30.0"/>
    <param name="lv" type="double" value="10.0"/>
    <param name="h" type="double" value="50.0"/>
    <param name="SPEED_SCALE" value="$(arg SPEED_SCALE)"/>
    <param name="HEADWAY_SCALE" value="$(arg HEADWAY_SCALE)"/>
    <param name="ego_vel_topic" value="$(arg ego_veltopic)"/>
    <param name="relative_vel_topic" value="$(arg relative_veltopic)"/>
    <param name="headway_topic" value="$(arg headway_topic)"/>
    <param name="use_lead_vel" value="false"/>
    <node pkg="onnx2ros" type="prompt_mode" name="controller" output="screen">

	<remap from="v_des" to="/cmd_vel"/>
   </node>
</launch>
